pkg/scheduler/server/server.go

```diff
@@ -153,7 +153,7 @@ func (s *Server) Run(ctx context.Context) error {
 	}
 	return concurrency.NewRunnerManager(
 		s.runServer,
-		s.runEtcd,
+		s.runEtcdCron,
```

Comment:
I liked this as is, as `runEtcd` since its starting the embedded etcd first, then initializes the etcd cron instance,  configures the etcd-cron with an etcd client, and starts the etcd cron scheduler. 

I think `runEtcd` encapsulates both parts

Reply:
We will support running EtcdCron without Etcd to allow a multi-tenant Etcd instance. When I was reading this, I though it was just Etcd.

Reply:
@JoshVanL any thoughts on this?

Reply:
If `runEtcdCron` encapsulates running both etcd and cron then this name is more correct, though I'd ask why these two long running processes are not distinct with `runETCD` and `runCron`.